src/scripts/domParsers/domUtils.ts

```diff
@@ -63,14 +160,42 @@ export module DomUtils {
 		Tags.video
 	];
 
-	export function removeElementsNotSupportedInOnml(doc: Document) {
+	/**
+	 * Given an HTML Document in string form, return an HTML Document in string form
+	 * with the attributes and the content between the HTML tags scrubbed, while preserving
+	 * document structure
+	 */
+	export function cleanHtml(contentInHtml: string): string {
```

Comment:
Can we call this sanitizeHtml()?  I think the name would describe the functionality a little better.